package info.isaaclee.lolgoitne.core.application.service.riot.exceptions

class SummonerNotFoundException: Exception("summoner_not_found") {
}
